hudi-common/src/main/java/org/apache/hudi/common/model/HoodieLogFile.java

```diff
@@ -60,7 +59,7 @@ public HoodieLogFile(FileStatus fileStatus) {
   public HoodieLogFile(Path logPath) {
     this.fileStatus = null;
     this.pathStr = logPath.toString();
-    this.fileLen = 0;
+    this.fileLen = -1;
```

Comment:
So the file len of 0 is very confusing it could be a legitimate case. I had to go and check whether file was indeed of length 0. 

Setting it to -1 makes it clear that this is not initialized and can't be used as is.